modules/cnf-configuring-workload-hints.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+:_content-type: CONCEPT
+[id="cnf-configuring-workload-hints_{context}"]
+= Configuring workload hints
+
+Workload hints simplify the fine-tuning of performance to industry sector settings. Instead of a “one size fits all” approach, workload hints can cater to usage patterns such as placing priority on:
+
+* Low latency
+* Real-time capability
+* Efficient use of power
+
+In an ideal world, all of those would be prioritized: in real life, some come at the expense of others. The Node Tuning Operator is now aware of the workload class and better able to meet the demands of a workload class. The cluster admin can now specify into which use case a workload falls. The Node Tuning Operator uses hints fine tune the performance settings.
+
+The environment in which an application is operating influences the choice of hints. In settings such as a core network, only minimal default power tuning is supported using CPU partitioning. In  distributed or Edge settings, latency is a higher priority, but some measures are still taken to optimize power consumption. For far edge settings, at a plane where where software-based radio is transmitted, ultra-low latency is the ultimate priority, and is achieved at the expense of power management.
+
+[cols="1,1,1,1",options="header"]
+|===
+    | Hint | Application | Environment | Description
+
+    | Default
```

Comment:
This column needs a header pointing to the Performance profile creator power mode

Reply:
I suggest preceding the table with a Note: "The following workload hints can be configured manually or created using the Performance Profile Creator. See "Creating a performance profile" for further information."